node_js/metrics/crashes_checks/services/ElasticSearchService.js

```diff
@@ -29,5 +33,43 @@ module.exports = {
     };
     const url = `/*-error-*/_count`;
     return api.post(url, body);
-  }
+  },
+  getTransactionMetrics: (projectName, environment) =>
+    api.post('/apm-*-transaction-*/_search?size=0', {
```

Comment:
This body is getting larger than expected. You could change the call to be similar to the environmentErrors: `api.post(url, body)` and segment the body into smaller pieces as you made with the `projectFilters` piece. Compose the body with all the small json pieces